docs/content/getting_started/_index.en.md

```diff
@@ -42,7 +42,7 @@ docker run -p 8080:8080 -v $PWD/models:/models -ti --rm quay.io/go-skynet/local-
 # └───────────────────────────────────────────────────┘
 
 # Try the endpoint with curl
-curl http://localhost:8080/v1/completions -H "Content-Type: application/json" -d '{
+curl http://localhost:8080/v1/chat/completions -H "Content-Type: application/json" -d '{
      "model": "your-model.gguf",
      "prompt": "A long time ago in a galaxy far, far away",
```

Comment:
this doesn't look right, the chat completion doesn't actually use a prompt, but it expects a list of messages

Reply:
@mudler Your right Ill fix it thank you!